nbformat/tests/test_sign.py

```diff
@@ -28,7 +28,16 @@ def setUp(self):
     
     def tearDown(self):
         shutil.rmtree(self.data_dir)
-    
+   
+    def test_invalid_db_file(self):
+        invalid_notary = sign.NotebookNotary(
+            db_file='invalid_db_file.db',
+            secret=b'secret',
+            data_dir='.',
```

Comment:
`.` means the working directory, which will be where you're running the tests from, not this directory. You can get the directory containing this file with:

```python
os.path.dirname(__file__)
```

Because the code we're testing is going to move and replace that file, it's probably a good idea to copy it to a temporary folder and use that copy for the tests - otherwise, we have to keep resetting the test file after running the tests.